.vscode/launch.json

```diff
@@ -64,5 +76,13 @@
 				"start",
 			]
 		},
+	],
+	"inputs": [
+		{
+			"type": "promptString",
+			"id": "functionalTestSuiteMethod",
+			"description": "The method name withing the functional test suite that you wish to debug.",
```

Comment:
This PR is great, I'm happy for us to be improving VSCode experience for server development. There's a little typo here.

Reply:
https://github.com/temporalio/temporal/pull/5618/files